upgrade/upgrade.go

```diff
@@ -61,6 +65,25 @@ func CreateUpgradeHandler(
 			return vm, err
 		}
 
+		bankKeeper.SetDenomMetaData(ctx, banktypes.Metadata{
```

Comment:
Could we get this extracted into a function like the rest of the logic!

```go
// ConfigureBankModule sets the bank metadata for the Noble Dollar.
func ConfigureBankModule(ctx context.Context, bankKeeper bankkeeper.Keeper) {}

Reply:
https://github.com/noble-assets/noble/pull/504/commits/bc275445b0388d49df68f000c766bfbef3ed267c